cms/templates/js/timed-examination-preference-editor.underscore

```diff
@@ -24,29 +61,5 @@
                 <p class='field-message'><%- gettext('Students see warnings when 20% and 5% of the allotted time remains. In certain cases, students can be granted allowances that give them extra time to complete the exam.') %></p>
```

Comment:
Udpate text to:

"Learners see warnings when 20% and 5% of the allotted time remains. You can grant learners extra time to complete the exam through the Instructor Dashboard."